docs/current/guides/194031-kubernetes.md

```diff
@@ -0,0 +1,222 @@
+---
+slug: /194031/kubernetes
+displayed_sidebar: "current"
+category: "guides"
+tags: ["kubernetes", "github", "eks", "dagger-cloud"]
+authors: ["Joel Longtine", "Gerhard Lazu", "Vikram Vaswani"]
+date: "2023-09-22"
+---
+
+# Run Dagger on Kubernetes
+
+## Introduction
+
+This guide outlines how to set up a Continuous Integration (CI) environment with the Dagger Engine on Kubernetes. It describes and explains the recommended architecture pattern and components, together with optional optimizations. It also describes a specific implementation using GitHub Actions, Amazon Elastic Kubernetes Service (EKS) and Karpenter.
+
+## Assumptions
+
+This guide assumes that you have:
+
+- A good understanding of how Kubernetes works, and of key Kubernetes components and add-ons.
+- A good understanding of how Dagger works. If not, [read the Dagger Quickstart](../quickstart/index.mdx).
+
+Readers wishing to implement the recommended architecture pattern with GitHub Actions and Amazon EKS, as described in the later section of this guide, should additionally have:
+
+- An Amazon EKS Kubernetes cluster with [cert-manager](https://cert-manager.io/), [Karpenter](https://karpenter.sh/), [Helm](https://helm.sh) and [GitHub Actions Runner Controller (ARC)](https://github.com/actions/actions-runner-controller) installed.
+- A GitHub account. If not, [sign up for a free GitHub account](https://github.com/signup).
+- A Dagger Cloud account. If not, [sign up for Dagger Cloud](https://dagger.io/cloud).
+
+## Architecture Patterns
+
+### Base pattern
+
+The base pattern consists of persistent Kubernetes nodes with ephemeral CI runners.
+
+The minimum required components are:
+
+- **Kubernetes cluster**, consisting of support nodes and runner nodes.
```

Comment:
I generally try to avoid formatting in bold, as the CSS automatically makes links bold, so bold text could be confused for a clickable link.

Reply:
I wanted these to stand out, but I get your point. Making the change now.

Reply:
Thanks for changing. I have reformatted them in italics to highlight them. LMK if that's OK, if not I will revert to "no formatting".